src/utils/exportOptions.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable no-unused-vars */
+export enum ExportOptions {
```

Comment:
A type with "download" | "clipboard" is better than an enum here (enums are not that great in general in Typescript)